dipy/tracking/metrics.py

```diff
@@ -758,71 +758,6 @@ def _extrap(xyz, cumlen, distance):
     return Lambda*xyz[ind]+(1-Lambda)*xyz[ind-1]
 
 
-def downsample(xyz, n_pols=3):
```

Comment:
To not break existing code, Instead of deleting this entirely, could we do something like the following? 

    downsample = set_number_of_points

Reply:
I just see your review, sorry, I duplicated your comment 😄